### PR TITLE
Serve to iOS simulator docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator](https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator)

File: src/app/learn/get-started/basics/serve-project/ios-simulator/index.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      </p>
+      <ul>
+        <li>
+          <p>
+            On your Mac, open Safari and navigate in the toolbar to <strong>Safari</strong>, <strong>Preferences</strong>, <strong>Advanced</strong> and enable <strong>Show Develop menu in menu bar</strong>.
+          </p>
+        </li>
+        <li>
+          <p>
+            Open <a href="http://preview.mobify.com">http://preview.mobify.com</a> in the iOS simulator.
```
Comment (Blackbaud-AlexKingman) [inline src/app/learn/get-started/basics/serve-project/ios-simulator/index.html:81]: Is this mobify step necessary? Why couldn't you just link directly to the SPA URL in the iOS toolbar?

File: src/app/learn/get-started/basics/serve-project/ios-simulator/index.html [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+      </p>
+    </li>
+    <li>
+      <p>
+        Set up the simulated device to allow local development.
+      </p>
+      <ul>
+        <li>
+          <p>
+            On your iOS device, navigate to <strong>Settings</strong>, <strong>Safari</strong> and turn off <strong>Prevent Cross-Site Tracking</strong>.
```
Comment (Blackbaud-AlexKingman) [inline src/app/learn/get-started/basics/serve-project/ios-simulator/index.html:39]: Should this say, "On your _simulated_ iOS device" ?

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Everything worked for me! Just a couple comments.

File: src/app/learn/get-started/basics/serve-project/ios-simulator/index.html (hunk 2 above)
Comment (blackbaud-johnly) [reply src/app/learn/get-started/basics/serve-project/ios-simulator/index.html:39]: Makes sense to me. I'll make the change.

File: src/app/learn/get-started/basics/serve-project/ios-simulator/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/learn/get-started/basics/serve-project/ios-simulator/index.html:81]: I have no idea. This is the guidance in the instructions, but I don't have a Mac to test this. Do you want me to change this? And if so, does the next bullet need to change too?

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator](https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator)

File: src/app/learn/get-started/basics/serve-project/ios-simulator/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/learn/get-started/basics/serve-project/ios-simulator/index.html]: I removed the mobify steps.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator](https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator](https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator)

**Review by Blackbaud-AlexKingman — APPROVED** [review]

File: src/app/learn/get-started/basics/serve-project/ios-simulator/index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      </ol>
+    </li>
+    <li>
+      <p>
+        Run <sky-code>skyux serve --browser safari</sky-code> to serve your SPA to Safari, and then use the link provided in the command line to view your SPA.
+      </p>
+    </li>
+    <li>
+      <p>
+        To allow console logging and general debugging, follow these steps:
```
Comment (blackbaud-paulah) [inline src/app/learn/get-started/basics/serve-project/ios-simulator/index.html:71]:
```suggestion
        To allow console logging and general debugging, follow these steps.
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator](https://developer.blackbaud.com/skyux?_sp.spa=pr-serve-in-ios-simulator)

**Review by blackbaud-paulah — APPROVED** [review]

**Pull request merged**